test.sh

```diff
@@ -48,8 +48,8 @@ SUBDIR='demo'
 
 # Test
 
-docker exec -i $SERVERID sh -c "emacs --batch --eval '(message (pp (+ 2 2)))' && cd /dir && \
-echo -ne \"\ntest\ntest\" | learn-ocaml-client set-options --server=http://localhost:8080 && \
+docker exec -i $SERVERID sh -c "emacs --batch --eval '(message (pp (+ 2 2)))' && cd /home/learn-ocaml/dir && \
+learn-ocaml-client init --server=http://localhost:8080 test test && \
```

Comment:
is there a reason why you put `test` twice here? (`test test`)

Reply:
The first is the nickname and the second one is the secret